components/brave_wallet/browser/swap_service.cc

```diff
@@ -302,11 +302,13 @@ void SwapService::GetPriceQuote(mojom::SwapParamsPtr swap_params,
   auto internal_callback =
       base::BindOnce(&SwapService::OnGetPriceQuote,
                      weak_ptr_factory_.GetWeakPtr(), std::move(callback));
+
   api_request_helper_.Request(
-      "GET",
+      net::HttpRequestHeaders::kGetMethod,
       GetPriceQuoteURL(std::move(swap_params),
                        json_rpc_service_->GetChainId(mojom::CoinType::ETH)),
-      "", "", true, std::move(internal_callback), Get0xAPIHeaders());
+      base::EmptyString(), base::EmptyString(), true,
```

Comment:
What is the purpose of base::EmptyString() here? It is designed to for cases when we need to return a reference to a string but don't have that string.
That's works of course, but will confuse every reader why not just double quotes.

Reply:
Reverted to using `""` instead of `base::EmptyString()`. See https://github.com/brave/brave-core/pull/17763/commits/d35af2a1940953b483dbad6736898d11274b544a.